cypress/e2e/hub/repositories.cy.ts

```diff
@@ -1,299 +1,224 @@
-    cy.verifyPageTitle(repositoryNameLocal);
-    cy.get('[data-cy="actions-dropdown"]').click();
-    cy.get('[data-cy="delete-repository"]').click();
-    cy.get('#confirm').click();
-    cy.get('button').contains('Delete repositories').click();
-    cy.verifyPageTitle(Repositories.title);
-    cy.get('[data-cy="text-input"]').type(repositoryNameLocal);
-    cy.get('.pf-v5-c-empty-state').should('be.visible');
-    cy.get('.pf-v5-c-empty-state').contains('No results found');
-  });
-});
+    cy.createHubRepository().then((repository) => {
+      cy.filterTableBySingleText(repository.name, true);
+      cy.clickTableRow(repository.name, false);
 
-describe('Repositories Remove collection', () => {
-  let collections: string[];
-  let namespace: string;
-  let repository: string;
-  let numCollections: number;
+      cy.verifyPageTitle(repository.name);
+      cy.get('[data-cy="actions-dropdown"]').click();
+      cy.get('[data-cy="delete-repository"]').click();
+      cy.get('#confirm').click();
+      cy.get('button').contains('Delete repositories').click();
 
-  afterEach(() => {
-    collections?.forEach((collection) => {
-      cy.galaxykit('-i collection delete ', namespace, collection, '1.0.0', repository);
-      cy.galaxykit('task wait all');
+      cy.verifyPageTitle(Repositories.title);
+      cy.filterTableBySingleText(repository.name, true);
+      cy.get('.pf-v5-c-empty-state').should('be.visible');
+      cy.get('.pf-v5-c-empty-state').contains('No results found');
     });
-    cy.galaxykit('task wait all');
-    cy.galaxykit('-i repository delete', repository);
-    cy.galaxykit('task wait all');
-    cy.galaxykit('-i namespace delete', namespace);
-  });
-
-  beforeEach(() => {
-    cy.hubLogin();
   });
 
-  it('it should add and then remove the collections', () => {
-    // fill data
-    numCollections = 3;
-    collections = [];
-    for (let i = 0; i < numCollections; i++) {
-      collections.push('hub_e2e_collection_' + randomString(5, undefined, { isLowercase: true }));
-    }
-
-    namespace = 'hub_e2e_appr_namespace' + randomString(5).toLowerCase();
-    repository = 'hub_e2e_appr_repository' + randomString(5);
-
-    cy.galaxykit(`repository create ${repository}`);
-    cy.galaxykit('task wait all');
-
-    cy.galaxykit(`distribution create ${repository}`);
-    cy.galaxykit('task wait all');
-
-    cy.galaxykit(`namespace create ${namespace}`);
-    cy.galaxykit('task wait all');
-
-    collections.forEach((collection) => {
-      cy.galaxykit(`collection upload ${namespace} ${collection}`);
-      cy.galaxykit('task wait all');
+  it('should copy CLI to clipboard', () => {
+    cy.createHubRepository().then((repository) => {
+      cy.filterTableBySingleText(repository.name, true);
+      cy.clickTableRow(repository.name, false);
+      cy.verifyPageTitle(repository.name);
+      cy.clickPageAction('copy-cli-configuration');
+      cy.get('[data-cy="alert-toaster"]').should('be.visible');
```

Comment:
Missing closing the alert.

Reply:
The alert should be closed as it can stay in the UI even when a page changes so it can negatively influence following tests by covering toolbar kebab.